.github/workflows/ci.yml

```diff
@@ -104,10 +105,14 @@ jobs:
              ../cmake
           sudo make -j $(nproc)
           sudo make install
+      - name: Install IfcConvert
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ifcconvert
```

Comment:
Hi, I don't think a debian package exists for ifcconvert. Even if it would exist, the aim is to use the code compiled in the process steps above (`sudo make install`) so that we actually test the committed code and not an old version in the package manager.